### PR TITLE
Bringing the layouts in line with spacing standards

### LINKED ISSUE #1396: General atomic element spacing guidlines
I collected some general spacing thoughts for spacing between molecules and organisms that are consistent across page types. I thought I would see if this helped and make sure we were all on the same page with these before we have more discussion

#### Standard spacing (no rule line)
screen shot 2016-02-01 at 1 00 32 pm
* 30px above 
 * This is true of mols/orgs under heroes as well
 * unless it is a text introduction under a breadcrumb which would be [handled like this](https://github.cfpb.gov/flapjack/Modules-V1/wiki/Breadcrumbs)
* 60px after
 * This is true when the mol/org is the last piece to a section. The next element should be a rule line or mol/org of a new section.
* This is true for the last mol/org on a page before the footer. *(there should always be 60px between the last thing on the page and the footer)*
* **These spacings are not additive! There should not be more than 60px between mol/orgs unless there is a rule line (see below)**

#### Spacing with rule line divider between sections
screen shot 2016-02-01 at 1 00 47 pm
* The rule line acts a mol/org without spacing of it's own 
* mol/org before rule line has standard 60px after, the rule line, and the mol/org that follows the rule has the standard 30px above. 

#### Spacing between multiple 25/75 molecules
screen shot 2016-02-01 at 1 00 53 pm
* The [specs for the 25/75](https://github.cfpb.gov/flapjack/Modules-V1/wiki/25-75-Image-&-Text) do not reflect this but we use this molecule commonly without rule lines on top and bottom. The spacing in the spec is still generally correct without the rules.
* An update that was made in build was allowing for 30px of space between 25/75 molecules where there were multiples of these in the same section stacked on top of each other. 
 * The last 25/75 molecule should still have 60px of space after. 
 * This spacing should not be used if the multiple 25/75 molecules are not part of the same section or are not intended to be grouped. 
 * I do not think any other molecules abide by this spacing structure. 

#### Other instances
* Sidebar mol/orgs abide by the same spacing structure and the slug headings are considered part of a given sidebar mol/org. 
* Full width text molecules follow this structure when the mol/org  above or below is not another full width text molecule. 
 * When multiple full width text molecules are next to each other the spacing should reflect the [web type hierarchy spacing standards](https://github.com/cfpb/design-manual/issues/269#issuecomment-157544901)

@ajbush and @schaferjh - do these guidelines follow your thinking for our general approach to spacing elements on all page types? Are there things  that are missing or need to be added? Do any of these sound incorrect?

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Fixes a number of spacing issues, specifically with the text-image molecules, the first block of the page on several templates and spacing at the end of pages.

Fixes #1396 and a lot of #1366 

## Review
- @jimmynotjim 
- @anselmbradford 
- @sebworks

### CODE REVIEW

**Comment by anselmbradford** [comment]
Code looks good :+1: How best to test this?

One trivial point, you could stack the classes that call super:
```
{{ super() }}
content__flush-bottom
```

File: cfgov/unprocessed/css/organisms/image-text-50-50-group.less [hunk 1]
```diff
@@ -16,13 +16,13 @@
     - cfgov-molecules
 */
 .o-image-text-50-50-group {
-    // TODO: Find another way to handle spacing
+    // TODO: Find another way to handle spacing here and in link-blob-group
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/image-text-50-50-group.less]: I've got an idea for this I'm testing on the homepage. Now that it's merged, I'm going to extend the info-unit stuff here.

**Comment by KimberlyMunoz** [comment]
Testing is going to be a pain. The best way I could see is to create templates with one molecule on each page and checking the spacing for the 25-75, 50-50, and link blob specifically.

File: cfgov/unprocessed/css/organisms/image-text-50-50-group.less (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/unprocessed/css/organisms/image-text-50-50-group.less]: Discussed offline We'll get this in another PR.

**Comment by jimmynotjim** [comment]
Am I the only one that can still test the demo pages? Built and tested there and it looks good! :+1:

**Comment by sebworks** [comment]
:

**Pull request merged**